### PR TITLE
Update authorization.rst

### PR DESCRIPTION
**Pull request by luizcmarin**
@markstory

### CODE REVIEW

File: en/tutorials-and-examples/cms/authorization.rst [hunk 1]
```diff
@@ -88,6 +88,12 @@ queries. Lets use bake to generate a basic policy:
 .. code-block:: bash
 
     bin/cake bake policy --type entity Article
+    
+    and
+    
+    bin/cake bake policy --type table Articles   (from cakephp v2 book)
```
Comment (markstory) [inline en/tutorials-and-examples/cms/authorization.rst:94]: Why did you need this policy? All the checks in the tutorial are against the entity.

Comment (luizcmarin) [reply en/tutorials-and-examples/cms/authorization.rst:94]: I don't need it, but without it the cake would generate an error. Looking at the cake v2 documentation, I saw this command line and ran it, so the cake worked, although there were other errors.

Comment (luizcmarin) [reply en/tutorials-and-examples/cms/authorization.rst:94]: > All the checks in the tutorial are against the entity.

I did not understand this sentence. Could you clarify it?

Comment (markstory) [reply en/tutorials-and-examples/cms/authorization.rst:94]: > I did not understand this sentence. Could you clarify it?

All of the tutorial authorization checks are made against an entity and not the table. Perhaps you were trying to use the application between generating the policy with bake and updating it with the correct methods?
